packages/terra-brand-footer/src/BrandFooter.scss

```diff
@@ -1,53 +1,118 @@
 :local {
   .brand-footer {
-    margin: none;
+    margin: 0;
   }
 
   .nav {
-    background-color: var(--terra-brand-footer-nav-background-color, #f4f4f4);
-    color: var(--terra-brand-footer-nav-color, #64696c);
-    display: block;
+    background-color: var(--terra-brand-footer-nav-background-color, #153d70);
+    color: var(--terra-brand-footer-nav-color, #8baaca);
+
+    display: flex;
+    flex-wrap: wrap;
     padding: 0.7142857142857143rem 1.4285714285714286rem;
+    width: 100%;
   }
 
-  .menu {
-    list-style: none;
-    margin: 0;
-    padding-left: 0;
-    padding-right: 0;
-  }
+  .nav-vertical {
+    align-content: stretch;
+    flex-direction: row;
+
+    .menu {
+      flex: 1 1 0%;
+      min-width: 240px;
+      padding-right: 1.4285714286rem;
+    }
+
+    .list-item {
+      display: block;
+      padding-bottom: 0.357142857142857rem;
+      padding-top: 0.357142857142857rem;
+    }
 
-  .list-item {
-    display: inline;
+    .list-header {
+      display: block;
+      padding-bottom: 0.7142857143rem;
+      padding-top: 1.4285714286rem;
+    }
   }
 
+  .nav-horizontal {
+    flex-direction: column;
 
-  .list-item:first-child::before {
-    content: '';
-    padding: 0;
+    .menu {
+      width: 100%;
+    }
+
+    .list-item {
+      display: inline-block;
+    }
+
+    .list-header {
+      display: inline-block;
+    }
+  
+    .list-item:first-child::before {
+      content: '';
+      padding: 0;
+    }
+
+    /* stylelint-disable selector-max-compound-selectors */
+    // A '/' isn't needed to separate an item from the header it follows
+    .list-header + .list-item::before {
```

Comment:
Do we need to apply these list item before styles to vertical displays as well?

Reply:
No because vertical links are each given their own line to clearly separate them from other links. Horizontal links must insert a `/` between each other to show their separation. That is what this style controls.